docs/organizations/managing-repositories.md

```diff
@@ -1,10 +1,12 @@
 # Managing repositories
 
-To see a list of the repositories that Codacy is monitoring, open the page **Repositories** under your organization.
+Users with the [necessary permissions on your Git provider](roles-and-permissions-for-organizations.md) can **add** repositories to Codacy to start analyzing them. The remaining organization members can then **follow** added repositories of their interest on Codacy.
+
+To see a list of the repositories you follow on Codacy, open the page **Repositories** under your organization. Across the application, Codacy calculates and displays data for the repositories on this list.
```

Comment:
We can see a list of repositories even in the overview dashboard, so I was thinking if saying `To see the full list of repositories you follow (...)` isn't better? 🤔

Reply:
Thanks for the input, @inesgomas! 👍 I tweaked the sentence.